trade-module/src/main/java/io/edpn/backend/trade/adapter/persistence/repository/MybatisSystemRepository.java

```diff
@@ -26,20 +28,28 @@ public interface SystemEntityMapper {
             @Result(property = "yCoordinate", column = "y_coordinate", javaType = Double.class),
             @Result(property = "zCoordinate", column = "z_coordinate", javaType = Double.class)
     })
-    Optional<SystemEntity> findById(UUID id);
+    Optional<MybatisSystemEntity> findById(UUID id);
 
     @Select("SELECT * FROM system WHERE name = #{name}")
     @ResultMap("systemResultMap")
-    Optional<SystemEntity> findByName(@Param("name") String name);
+    Optional<MybatisSystemEntity> findByName(@Param("name") String name);
 
     @Insert("INSERT INTO system (id, name, elite_id, x_coordinate, y_coordinate, z_coordinate) " +
             "VALUES (#{id}, #{name}, #{eliteId}, #{xCoordinate}, #{yCoordinate}, #{zCoordinate})")
-    void insert(SystemEntity system);
+    void insert(MybatisSystemEntity system);
 
     @Update("UPDATE system SET name = #{name}, elite_id = #{eliteId}, x_coordinate = #{xCoordinate}, " +
             "y_coordinate = #{yCoordinate}, z_coordinate = #{zCoordinate} WHERE id = #{id}")
-    void update(SystemEntity system);
+    void update(MybatisSystemEntity system);
 
     @Delete("DELETE FROM system WHERE id = #{id}")
     void delete(UUID id);
+
+    @Select({"SELECT * FROM system",
+            "WHERE 1 = 1",
```

Comment:
Is there a reason that we need 1=1?

Reply:
I some old thing I sometimes forget I don't need to add,

used to have a stringbuilder query system and each optional line started with `AND` so you needed an initial where that always was true.

can be removed here

Reply:
converted to script but would likely be improved more as a stringbuilder